### ISSUE TITLE
univ/2 giving false

### ISSUE THREAD

**Issue by josd** [infradig/trealla#63]
We currently get
```
Trealla ProLog (c) Infradig 2020, v1.2.59-1-g8e57
?- a(X) =.. [Y|Z].
false.
```
whereas we expect
```
?- a(X) =.. [Y|Z].
Y = a,
Z = [X].
```

**Comment by infradig** [comment]
Really? ok. Good news i've solved #61 latest incarnation! Upload later
today when I do this one.

**Comment by josd** [comment]
Very well and thanks a lot!
